### PR TITLE
Add hub services page

### PR DESCRIPTION
**Pull request by jnywong**
Closes https://github.com/2i2c-org/meta/issues/1197

### CODE REVIEW

**Review by Gman0909 — COMMENTED** [review]
I think this will work nicely!

**Comment by jnywong** [comment]
Cool! I have added some images and video to make the page a bit more lively. I've noticed some overlap with the https://2i2c.org/platform/ page, but I think this is safe to try for now (I can envision re-working those pages more as we refine our product offering 💪)

File: content/hub-services/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+aliases:
+  - /hub-services/
+
+
+# Your landing page sections - add as many different content blocks as you like
+sections:
+  - block: markdown
+    id: hub-services-intro
+    content:
+      title: Your interactive computing hub, the way you want it
```
Comment (Gman0909) [inline content/hub-services/index.md]: This is rendering a bit funky at the current width. Maybe changing the language a bit to "Your hub, the way you want it" would help

image

File: content/hub-services/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+          {{< video src="videos/jupyterhub-admin.mp4">}}
+
+          <figcaption>
+              Powered by <a href="https://jupyterhub.readthedocs.io"><img src="/images/logos/project/jupyterhub.svg" /></a>
+          </figcaption>
+        </figure>      
+
+  - block: features
+    content:
+      title: Compute
```
Comment (Gman0909) [inline content/hub-services/index.md:41]: Rendering poorly in dark mode

image

**Review by Gman0909 — COMMENTED** [review]
The new page looks fantastic! Couple of small glitches commented.

File: content/hub-services/index.md (hunk 2 above)
Comment (jnywong) [reply content/hub-services/index.md:41]: Nice catch! I tried to hack something that I shouldn't have lol

Comment (jnywong) [reply content/hub-services/index.md:41]: Resolved in https://github.com/2i2c-org/2i2c-org.github.io/pull/295/commits/ff4e4977358f5dfd20eb8eced8966e3573682ff8

File: content/hub-services/index.md (hunk 1 above)
Comment (jnywong) [reply content/hub-services/index.md]: Resolved in https://github.com/2i2c-org/2i2c-org.github.io/pull/295/commits/49d0786997c27ce1d8f8133080c2bb251d68ba68

File: content/hub-services/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        Hubs can be configured with a choice of persistent home directories and different levels of read-write permissions (admins only, or everyone).
+
+        Additionally, object storage can be configured for access to scratch, persistent, or pre-existing buckets, as needed.
+
+  - block: markdown
+    content:
+      title: Sharing
+      subtitle: ""
+      text: |
+        Learning and discovery through interactive cloud computing is more powerful when shared.
```
Comment (Gman0909) [inline content/hub-services/index.md]: Can we please change the "is" to "are"? My fault, it was my mistake on the text I provided

Comment (jnywong) [reply content/hub-services/index.md]:
```suggestion
        Learning and discovery through interactive cloud computing are more powerful when shared.
```

**Review by Gman0909 — APPROVED** [review]
Amazing work, thanks @jnywong

**Pull request merged**